ccs-scale-infra-shared/terraform/environments/sbx1/main.tf

```diff
@@ -31,3 +31,9 @@ module "deploy" {
   aws_account_id = data.aws_ssm_parameter.aws_account_id.value
   environment    = local.environment
 }
+
+module "cloudtrail" {
```

Comment:
Ahh I was wondering why `terraform plan` on SBX2 was proposing no changes.. I'll copy in now I guess this is intentional (i.e. we only need this auditing on higher environments)?

Reply:
I thought so - I assume it's chargeable infra that's not required on the other environments. I did think about building it into another config - which I could look at doing still (that's why they are there I guess) - 'configs/deploy-all-cloudtrail' or something maybe?